pkg/standalone-utils/test/foundry/WeightedLPOracle.t.sol

```diff
@@ -247,6 +257,47 @@ contract WeightedLPOracleTest is BaseVaultTest, WeightedPoolContractsDeployer {
         oracle.computeTVLGivenPrices(prices);
     }
 
+    function testGetUptimeFeed() public {
+        (IWeightedPool pool, ) = createAndInitPool(2);
+        (WeightedLPOracleMock oracle, ) = deployOracle(pool);
+
+        assertEq(address(oracle.getSequencerUptimeFeed()), address(uptimeFeed), "Wrong uptime feed");
+    }
+
+    function testGetUptimeGracePeriod() public {
+        (IWeightedPool pool, ) = createAndInitPool(2);
+        (WeightedLPOracleMock oracle, ) = deployOracle(pool);
+
+        assertEq(oracle.getUptimeGracePeriod(), UPTIME_GRACE_PERIOD, "Wrong uptime grace period");
+    }
+
+    function testMalfunctioningUptimeFeed() public {
+        (IWeightedPool pool, ) = createAndInitPool(2);
+        (WeightedLPOracleMock oracle, ) = deployOracle(pool);
+        uptimeFeed.setRevertLatestRoundData(true);
+
+        vm.expectRevert(ISequencerUptimeFeed.SequencerFeedUnavailable.selector);
+        oracle.latestRoundData();
+    }
+
+    function testUptimeSequencerDown() public {
+        (IWeightedPool pool, ) = createAndInitPool(2);
+        (WeightedLPOracleMock oracle, ) = deployOracle(pool);
+        uptimeFeed.setLastRoundData(1, 0);
+
+        vm.expectRevert(ISequencerUptimeFeed.SequencerDown.selector);
+        oracle.latestRoundData();
+    }
+
+    function testGracePeriodNotOver() public {
+        (IWeightedPool pool, ) = createAndInitPool(2);
+        (WeightedLPOracleMock oracle, ) = deployOracle(pool);
+        uptimeFeed.setStartedAt(block.timestamp - 100);
+
+        vm.expectRevert(ISequencerUptimeFeed.GracePeriodNotOver.selector);
+        oracle.latestRoundData();
+    }
```

Comment:
WDYT about moving these tests to a base layer?

`createAndInit` would be overridden anyway, so each specific test would run its own variant properly. Do you think it's worth it?